index.bs

```diff
@@ -119,6 +119,19 @@ To <dfn>Construct an AmbientLightSensor Object</dfn> the user agent must invoke
 The <a attribute for="AmbientLightSensor">illuminance</a> attribute of the {{AmbientLightSensor}}
 interface represents the <a>current light level</a>.
 
+Use Cases and Requirements {#usecases-requirements}
+=========
+
+1. A Web application gradually updates document style based on light level changes.
+1. A Web application provides input for a smart home system to control lighting.
+1. A Web aplication checks whether light level at work space is sufficient, based on occupational
```

Comment:
> 10 Lux resolution that is guaranteed by sensor manufacturers, should be enough to measure illuminance for workplace.

Is calibration OK, though? i.e. is 300 Lux measured on a device _really_ 300 ± 10 Lux? That is, resolution is relative, not absolute.